internal/scorecard/alpha/scorecard.go

```diff
@@ -0,0 +1,117 @@
+// Copyright 2020 The Operator-SDK Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package alpha
+
+import (
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
+	"strings"
+
+	"github.com/operator-framework/operator-sdk/version"
+	"github.com/sirupsen/logrus"
+	"gopkg.in/yaml.v2"
+	"k8s.io/apimachinery/pkg/labels"
+)
+
+var (
+	Log = logrus.New()
+)
+
+type ScorecardFlags struct {
+	Config       string
+	Bundle       string
+	Selector     string
+	ListAll      bool
+	OutputFormat string
+	Kubeconfig   string
+}
+
+// RunTests executes the scorecard tests as configured
+func RunTests(flags ScorecardFlags) error {
+	scConfig, err := lookupConfig(flags)
+	if err != nil {
+		return err
```

Comment:
`fmt.Errorf` these errors with some context for where they came from

Reply:
refactored this code to take your suggestion.